submissions/MarharytaBoichenko/html_css_popup/index.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML-CSS-Popup</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/1.0.0/modern-normalize.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Merriweather:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/main.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="header">
+        <nav class="nav">
+          <ul class="nav__list list">
+            <li class="nav__item">
+              <a href="#" class="link nav__link">Mail</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="link nav__link">Images</a>
+            </li>
+          </ul>
+        </nav>
+        <ul class="account-block list">
+          <li class="account-block__item">
+            <input
+              type="checkbox"
+              class="account-block__checkbox"
+              name="menu"
+              id="popup"
+            /><label for="popup" class="account-block__menu"></label>
+            <div class="menu">
+              <ul class="menu__list list">
+                <li class="menu__item">
+                  <a href="#apps" class="menu__link"
+                    ><img
```

Comment:
Please format this file correctly, e.g. `<a href="#apps" class="menu__link">`   <br /> `<img...`

Reply:
(( I can`t find  because of which rule  of  prettier it is (  it seems  I check  all in my  prettier file

Reply:
Prettier PG https://prettier.io/playground/ looks working well. 
![image](https://user-images.githubusercontent.com/21179601/183495016-b135606a-b23f-44a5-abc1-52f73ac63bb3.png)
Try to find information about this issue and play with the settings of HTML formatting rules.

Reply:
but  not this, I  can`t  do anything with  it
https://www.dropbox.com/s/2hqhibiiazxrv59/%D0%A1%D0%BD%D0%B8%D0%BC%D0%BE%D0%BA%20%D1%8D%D0%BA%D1%80%D0%B0%D0%BD%D0%B0%202022-08-09%20%D0%B2%2016.53.14.png?dl=0

Reply:
What I saw in your screenshot it is exactly what should be. Self-closing tags should look like this